src/main/resources/db/changelog/changes/04-create-roles-table.yaml

```
@@ -0,0 +1,43 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-roles-table
+      author: chernonog
+      changes:
+        - createTable:
+            tableName: roles
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: name
+                  type: ENUM('ROLE_ADMIN','ROLE_USER','ROLE_GUEST')
+                  constraints:
+                    nullable: false
+                    unique: true
+  - changeSet:
+      id: insert-values-into-roles-table
+      author: chernonog
+      changes:
+        - insert:
+            tableName: roles
+            columns:
+              - column:
+                  name: name
+                  value: ROLE_ADMIN
+        - insert:
+            tableName: roles
+            columns:
+              - column:
+                  name: name
+                  value: ROLE_USER
```

It would also be better to remove this everywhere
```suggestion
      changes:
        - insert:
            tableName: roles
            columns:
              - column:
                  name: name
                  value: ADMIN
        - insert:
            tableName: roles
            columns:
              - column:
                  name: name
                  value: USER
```

and add ROLE_ in getAuthorities() or getAuthority()

---

thanks )